### PR TITLE
Document AA - AMR integration

### PR DESCRIPTION
**Pull request by ericbottard**
Fix https://github.com/pivotal/acc-release/issues/613

This applies to 1.7 only (main)

### CODE REVIEW

File: application-accelerator/provenance.hbs.md [hunk 1]
```diff
@@ -12,4 +16,20 @@ The `Provenance` transform provides traceability and visibility into the generat
...
\ No newline at end of file
+For more information on the structure of the file and how to enable application bootstrapping provenance, see [Provenance transform](creating-accelerators/transforms/provenance.hbs.md).
+
+## Integration with AMR
+Application Accelerator also closely integrates with the [Artifact Metadata Repository](../scst-store/amr/overview.hbs.md) (AMR).
+
+Every time an application is generated with an accelerator, an event is sent to the AMR store that
+roughly captures the same set of information as described in the paragraph above.
+
+The relevant AMR data models are [AppAcceleratorRun](../scst-store/amr/data-model-and-concepts.hbs.md#appacceleratorruns) (one for each invocation, also includes version information
```
Comment (kcoriordan) [inline application-accelerator/provenance.hbs.md:27]: what does 'one for each invocation' mean here?
If you could put it in sentence structure here, I can update?

File: application-accelerator/provenance.hbs.md [hunk 2]
```diff
@@ -12,4 +16,20 @@ The `Provenance` transform provides traceability and visibility into the generat
...
+about which accelerator was used) and [AppAcceleratorFragments](../scst-store/amr/data-model-and-concepts.hbs.md#appacceleratorfragments) (zero to N, tracks which fragments were used
```
Comment (kcoriordan) [inline application-accelerator/provenance.hbs.md:28]: What does 'zero to N' mean here?
Can you put it in a sentence, it seems like it lacks context here?

File: application-accelerator/provenance.hbs.md (hunk 1 above)
Comment (ericbottard) [reply application-accelerator/provenance.hbs.md:27]: I'm not sure I understand the question. It means there is one `AppAcceleratorRun` for each invocation of an accelerator (ie "a run")

File: application-accelerator/provenance.hbs.md (hunk 2 above)
Comment (ericbottard) [reply application-accelerator/provenance.hbs.md:28]: zero to N means there is between 0 and some number N instances of those, N being the number of fragments used by the accelerator. There is one instance of AppAcceleratorFragment for each (named) fragment used by the accelerator being run.

Hope that helps

**Pull request merged**